spec/features/editing_content/edit_edition_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-RSpec.feature "Edit an edition" do
-  scenario do
+RSpec.describe "Edit an edition" do
+  it do
```

Comment:
These scenario blocks seem to be the biggest casualty of this switch over hmm - I guess it was always a bit of a hack that we had empty descriptions.

We've also got the minor annoyance of convention to deal with too, for example: https://gds-way.cloudapps.digital/standards/testing-with-rspec.html#example-integration-test - although that has the oddity of `Rspec.scenario` - and then in the unit test above we've got a subject 🤦‍♂